### ISSUE TITLE
Typescript type support

### ISSUE THREAD

**Issue by mentos1386** [marcbachmann/opentelemetry-node-metrics#10]
Hey, awesome project. One thing missing are typescript type definitions. It looks the code base is quite small, would it be worth to "rewrite" in to typescript?

I guess the only type definition missing is for the export itself. That could be written as a `d.ts` definition without any rewrite. As the change itself is very small.

**Comment by djMax** [comment]
I rewrote this here for the latest otel apis, latest node APIs and in TS: https://github.com/sesamecare/opentelemetry-node-metrics

**Comment by fullammo** [comment]
Great work boys! @mentos1386 @djMax  Thanks for your efforts. will come back with my feedback as soon as i'm able to try it out in action! Btw, shouldn't this be part of the https://github.com/open-telemetry/opentelemetry-js-contrib repository? Seems like a great addition.

**Comment by mentos1386** [comment]
@fullammo yes, it would be nice if it would be in opentelemetry-js-contrib. There has been some work done on that already, but nothing completed https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1106
